stable/fairwinds-insights/templates/_env.yaml

```diff
@@ -36,7 +36,7 @@ env:
 {{- with .Values.installationCodeSecret }}
 - name: INSTALLATION_CODE
   secretKeyRef:
-    name: {{ .Values.installationCodeSecret }}
+    name: {{ . | quote }}
```

Comment:
aah i thought about this ... but thought the other one had same meaning

Reply:
I think you could still do `$.Values.installationCodeSecret`